app/app.js

```diff
@@ -11,8 +12,21 @@ const routes = require('./routes');
 
 const strategy = new Auth0Strategy(
   config.auth0,
-  (issuer, audience, profile, callback) => {
-    return callback(null, profile._json);
+
+  function(req, issuer, audience, profile, accessToken, refreshToken, params, callback) {
+    const management_client = new ManagementClient({
+      token: params.id_token,
+      domain: config.auth0.domain,
+    });
+
+    management_client
+      .getUser({ id: profile.id })
```

Comment:
This call is already made in Auth0 in the [Multifactor-Google-Authenticator-Do-Not-Rename rule](https://manage.auth0.com/#/rules/rul_E68KkMg1XAKuwlYz). It would be more efficient to add the authorization groups data to the user profile in the rule, rather than repeating the same HTTP request here in the proxy.